components/Volunteer/VolunteerOpportunityDetail.tsx

```diff
@@ -9,10 +10,13 @@ export interface VolunteerOpportunityProps {
 export const VolunteerOpportunityDetail = ({ opportunity }: VolunteerOpportunityProps) => (
   <StyledDetails>
     <StyledRoleTitle>
-      {opportunity.RoleTitle}
+      <span>{opportunity.RoleTitle}</span>
     </StyledRoleTitle>
     <StyledDescriptionBody>
       {opportunity.Description}
+      <div className='btn'>
```

Comment:
minor: To be more consistent with the rest of the styling, this could be a 
```
const StyledSignUpButtonContainer = styled('div')({ textAlign: 'end' })
```